src/test/java/seedu/address/storage/JsonAdaptedMemberTest.java

```diff
@@ -1,9 +1,9 @@
 package seedu.address.storage;
```

Comment:
I think all the variables which are named person in this file should be renamed to member.

Reply:
Oops, thanks